dnf/conf/config.py

```diff
@@ -471,11 +471,11 @@ class RepoConf(BaseConfig):
     """Option definitions for repository INI file sections."""
 
     def __init__(self, parent, section=None, parser=None):
-        masterConfig = parent._config if parent else libdnf.conf.ConfigMain()
-        super(RepoConf, self).__init__(libdnf.conf.ConfigRepo(masterConfig), section, parser)
+        parentConfig = parent._config if parent else libdnf.conf.ConfigMain()
```

Comment:
I'm doing the rename `masterConfig` -> `mainConfig` in libdnf and I think for consistency it should be named the same here. (even though it's a private attribute so it doesn't matter that much)

Reply:
OK, I've changed it.